apps/HelloBaremetal/cmake-twice/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eo pipefail
+
+# Prerequisite :
+#     Halide is installed system-wide in your host machine
+#      or environmental variable `HALIDE_ROOT` is set to point your own Halide installation
```

Comment:
CMake already honors the `Halide_ROOT` and `CMAKE_PREFIX_PATH` environment variables, so there's no need to handle them specially here.

Reply:
Fixed.